### PR TITLE
NO-ISSUE: Remove unused functions

### PR DESCRIPTION
**Pull request by nmagnezi**
## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: pkg/conversions/conversions.go [hunk 1]
```diff
@@ -18,10 +18,6 @@ func GibToMib(gib int64) int64 {
 	return gib * int64(units.KiB)
 }
 
-func MibToGiB(mib int64) int64 {
```
Comment (danielerez) [inline pkg/conversions/conversions.go:21]: these helper funcs are actually nice to have. maybe worth keeping them.

File: pkg/conversions/conversions.go [hunk 2]
```diff
@@ -38,10 +34,6 @@ func BytesToMib(bytes int64) int64 {
 	return bytes / int64(units.MiB)
 }
 
-func GbToMib(gb int64) int64 {
```
Comment (danielerez) [inline pkg/conversions/conversions.go:41]: same

File: pkg/conversions/conversions.go (hunk 1 above)
Comment (nmagnezi) [reply pkg/conversions/conversions.go]: Done

File: pkg/conversions/conversions.go (hunk 2 above)
Comment (nmagnezi) [reply pkg/conversions/conversions.go]: Done

**Pull request merged**